task-sdk/pyproject.toml

```diff
@@ -226,3 +228,8 @@ pythonpath = "tests"
 # Keep temporary directories (created by `tmp_path`) for 2 recent runs only failed tests.
 tmp_path_retention_count = "2"
 tmp_path_retention_policy = "failed"
+
+[tool.airflow]
```

Comment:
This is a nice way to keep track which shared distributions are used by the "using" distribution.